mentat/command/commands/search.py

```diff
@@ -21,19 +23,57 @@ async def apply(self, *args: str) -> None:
             stream.send(str(e), color="red")
             return
 
+        cumulative_tokens = 0
         for i, (feature, score) in enumerate(results, start=1):
+            i_str = f"{i:3}"
+            score_str = f"{score:.3f}"
+
             label = feature.ref()
-            if label.startswith(str(session_context.cwd)):
-                label = label[len(str(session_context.cwd)) + 1 :]
+            label = label.removeprefix(str(session_context.cwd) + "/")
             if feature.name:
```

Comment:
I personally dislike having the feature.name tacked onto the end of the results; on the one hand, I understand why it's useful (especially when including to know what you're including), but it's also way too long, dense, and unreadable. I think we should either clean it up or remove it. Try using /search with and without it to see what I mean.

Reply:
I agree